ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/prior_box.cpp

```diff
@@ -83,6 +83,36 @@ namespace ngraph
                         axes)};
                 }
 
+                OutputVector prior_box_clustered(const Node& node)
+                {
+                    using PriorBoxClustered = default_opset::PriorBoxClustered;
```

Comment:
I'm not sure this `using` is a good idea, especially I can see only one usage of `default_opset::PriorBoxClustered` here.